wilkes/roster.json

```diff
@@ -169,7 +169,11 @@
       "materials": [
         {
             "type": "cv"
-        }
+        },
+        {
+          "type": "video",
+          "url": "https://youtu.be/Ea4X_kf7-XQ"
+      }
```

Comment:
@orlasinclair 
2 things
1. Be careful about the indentation of you curly braces please
2. We only need the endpoint of the video, not the full url, can you please change to "url": "Ea4X_kf7-XQ"